GramAddict/core/filter.py

```diff
@@ -95,9 +98,10 @@ def check_profile(self, device, username):
             if field_min_potency_ratio is not None and (
                 int(followings) == 0
                 or followers / followings < float(field_min_potency_ratio)
+                or followers / followings > field_max_potency_ratio
```

Comment:
Please, ensure that `field_max_potency_ratio` is float. Even though the suggested value in json is a number, but some user could put a string, eg "0.3" instead. This would crash.